torchvision/ops/deform_conv.py

```diff
@@ -19,8 +19,11 @@ def deform_conv2d(
     dilation: Tuple[int, int] = (1, 1),
     mask: Optional[Tensor] = None,
 ) -> Tensor:
-    """
-    Performs Deformable Convolution, described in Deformable Convolutional Networks
+    r"""
+    Performs Deformable Convolution v2, described in 
```

Comment:
Lint is complaining about the training space in this line.

Reply:
Fixed it!